.github/workflows/cover.yml

```diff
@@ -0,0 +1,31 @@
+name: Test Coverage
+on:
+  pull_request:
+    branches:
+      - "*"
+jobs:
+  test:
+    strategy:
+      matrix:
```

Comment:
optional_ matrix is not needed, when all variables only have a single entry

Reply:
I like that it shows it in the checks block, so we always get the visual confirmation about the go version.